pkg/config/common.go

```diff
@@ -21,9 +34,9 @@ var (
 			"apis/v1alpha1",
 			"apis/v1beta1",
 		},
-		Controller: []string{
```

Comment:
Should we also keep the default value of `Controller` for backward compatibility?